scripts/cnn_variant_wdl/cram2model.wdl

```diff
@@ -0,0 +1,340 @@
+        mkdir "./tensors/"
+
+        java -Djava.io.tmpdir=tmp -jar ${gatk_jar} \
+        CNNVariantWriteTensors \
+        -R ${reference_fasta} \
+        -V ${input_vcf} \
+        -truth-vcf ${truth_vcf} \
+        -truth-bed ${truth_bed} \
+        -tensor-type ${tensor_type} \
+        -output-tensor-dir "./tensors/" \
+        -bam-file ${input_bam}
+        
+        tar -czf "tensors.tar.gz" "./tensors/"
+    }
+
+    output {
+      File tensors = "tensors.tar.gz"
+    }
+    runtime {
+        docker: "samfriedman/p3"
+        memory: "3 GB"
+        disks: "local-disk " + disk_size + " HDD"
+    }
+
+}
+
+
+task TrainModel {
+    Array[File] tar_tensors
+    String output_prefix
+    String tensor_type
+    File gatk_jar
+    Int disk_size
+    Int epochs
+    command {
+        for tensors in ${sep=' ' tar_tensors}  ; do
+            tar -xzf $tensors 
+        done
+
+        java -Djava.io.tmpdir=tmp -jar ${gatk_jar} \
+        CNNVariantTrain \
+        -input-tensor-dir "./tensors/" \
+        -model-name ${output_prefix} \
+        -image-dir "./" \
+        -tensor-type ${tensor_type} \
+        -epochs ${epochs}
+    }
+
+    output {
+        File model_json = "${output_prefix}.json"
+        File model_hd5 = "${output_prefix}.hd5"
+        File roc_png = "per_class_roc_${output_prefix}.png"
+        File training_png = "metric_history_${output_prefix}.png"
+    }
+
+    runtime {
+        docker: "samfriedman/p3"
+        memory: "16 GB"
+        disks: "local-disk " + disk_size + " HDD"
+
+#      docker: "samfriedman/gpu"
+#      gpuType: "nvidia-tesla-k80" 
+#      gpuCount: 1 
+#      zones: ["us-central1-c"]
+#      memory: "16 GB"
+#      disks: "local-disk 400 HDD"
+#      bootDiskSizeGb: "16" 
+    }
+}
+
+
+task MergeVCFs {
+    Array[File] input_vcfs
+    String output_prefix
+    File picard_jar
+    command {
+        java -jar ${picard_jar} \
+        MergeVcfs \
+        INPUT=${sep=' INPUT=' input_vcfs} \
+        OUTPUT=${output_prefix}_hc4_merged.vcf.gz
+    }
+
+    output {
+        File output_vcf = "${output_prefix}_hc4_merged.vcf.gz"
+    }
+
+    runtime {
+        docker: "broadinstitute/genomes-in-the-cloud:2.1.1"
+        memory: "16 GB"
+        disks: "local-disk 400 HDD"
+    } 
+}
+
+task SamtoolsMergeBAMs {
+    Array[File] input_bams
+    String output_prefix
+    File picard_jar
+    command {
+        samtools merge ${output_prefix}_bamout.bam ${sep=' ' input_bams} 
+    }
+
+    output {
+        File bamout = "${output_prefix}_bamout.bam"
+    }
+
+  runtime {
+    docker: "broadinstitute/genomes-in-the-cloud:2.1.1"
+    memory: "16 GB"
+    disks: "local-disk 400 HDD"
+  }    
+}
+
+task SplitIntervals {
+    File picard_jar
+    Int scatter_count
+    File intervals
+
+    command {
+        set -e
+        Picard_Jar=${picard_jar}
+        mkdir interval-files
+        java -Xmx6g -jar $Picard_Jar IntervalListTools I=${intervals} O=interval-files SCATTER_COUNT=${scatter_count}
```

Comment:
What's the original suffix on the output?

Reply:
switched to gatk scattering